zulipterminal/core.py

```diff
@@ -115,6 +116,26 @@ def search_messages(self, text: str) -> None:
         if focus_position >= 0 and focus_position < len(w_list):
             self.model.msg_list.set_focus(focus_position)
 
+    def mute_unmute_streams(self, button: Any) -> None:
+        popup = PopUpConfirmation(self, button)
+        self.gap = urwid.Divider()
+        type_of_action = "unmuting" if button.stream_id in \
+            self.model.muted_streams else "muting"
+        question = urwid.Text(("bold", "Confirm " + type_of_action +
+                               " of stream '" + button.caption+"' ?"),
+                              "center")
+        prompt = urwid.LineBox(
+            urwid.ListBox(
+                urwid.SimpleFocusListWalker(
+                    [question, self.gap, popup]
+                )))
+
+        self.loop.widget = urwid.Overlay(
+            prompt, self.view,
+            align=("relative", 20),
```

Comment:
Aligning it relative to the window's size makes it look a little misplaced...

This is how it looks for me when the window is its full size:
![image](https://user-images.githubusercontent.com/44770317/59187553-5429f500-8b93-11e9-8721-e341f0da5aa3.png)

And this is how it looks when it's smaller:
![image](https://user-images.githubusercontent.com/44770317/59187686-9e12db00-8b93-11e9-9556-e43fff1ee643.png)